src/python/Python-2.7.10/setup.py

```diff
@@ -831,9 +834,12 @@ def detect_modules(self):
                                          [ '/usr/sfw/lib' ] )
         else:
             ssl_libs = find_library_file(self.compiler, 'ssl',lib_dirs,
-                                     ['/usr/local/ssl/lib',
-                                      '/usr/contrib/ssl/lib/'
-                                     ] )
+                [
+                    # OpenSSL libraries for macOS are used from Homebrew.
```

Comment:
I think that we should also mention here that order is important... and maybe this should be an inline comment to make it clear that it only refers to the line below... and not the the other files